src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -296,7 +296,16 @@ function MoneyRequestView({
                             interactive={canEditMerchant}
                             shouldShowRightIcon={canEditMerchant}
                             titleStyle={styles.flex1}
-                            onPress={() => Navigation.navigate(ROUTES.EDIT_REQUEST.getRoute(report.reportID, CONST.EDIT_REQUEST_FIELD.MERCHANT))}
+                            onPress={() =>
+                                Navigation.navigate(
+                                    ROUTES.MONEY_REQUEST_STEP_MERCHANT.getRoute(
+                                        CONST.IOU.ACTION.EDIT,
+                                        CONST.IOU.TYPE.REQUEST,
+                                        transaction?.transactionID ?? '',
+                                        report.reportID,
```

Comment:
remove backto praams because of the error (more [here](https://github.com/Expensify/App/pull/36287))